gson/src/test/java/com/google/gson/TypeAdapterTest.java

```diff
@@ -73,7 +84,7 @@ public Integer read(JsonReader in) {
   }
 
   private static final TypeAdapter<String> adapter =
-      new TypeAdapter<>() {
+      new TypeAdapter<String>() {
```

Comment:
I think this is not needed since we compile the test code with Java 11.

Though I think IDEs sometimes have problems detecting this, and erroneously assume Java 7 is used for test compilation as well.